js/__fixtures__/contexts/odrl.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Same could be said for this one as what was left in example-v1.json

Reply:
this file a dependency of examples/v1 see https://github.com/w3c-ccg/security-vocab/pull/80#discussion_r541010434

Reply:
I removed these, so we only rely on citizenship